app/views/camp/add_new.php

```diff
@@ -0,0 +1,38 @@
+<h2>Add a new camp</h1>
+<?php
+
+if ($added !== null) {
+    if ($added == true) {
+        echo "Added successfully";
+    }
+    else {
+        echo "Submission failed";
+    }
+    echo "<br><br>";
+}
+?>
+<form action="" method="POST">
+    
+        <legend>Basic details about the Camp</legend>
+        <label for="name">Name: </label><input id="name" name="name" required /><br>
+        <label for="organisation_id">Organisation: </label><input id="organisation_id" name="organisation_id" required /><br/>
```

Comment:
We can't use organisation id directly. 
@nisargjhaveri We seriously need to create a auto complete widget. Thoughts??